### DISCUSSION TITLE
Player switch language

### DISCUSSION THREAD

**Discussion by HYDeath** [Zrips/CMI#6387] [Feature Requests]
Hello author, I think there is a function, I don't know whether this can be realized, which allows players to freely choose the language of the plug-in, and the language is only displayed by this player, and other players will not be affected.

**Comment by Zrips** [comment]
Will keep this in mind but with current system it would be some what tedious to define which locale we should check for needed line which would depend on specific player, especially when it comes to mass messages. Maybe in the future.
